runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -324,7 +325,10 @@ ILogData read(long address, @NonNull ReadOptions options) {
     private boolean cacheReadRequest(ReadOptions options) {
         if (runtime.getParameters().isCacheDisabled() && options.isClientCacheable()) {
             // TODO(Maithem): We can remove this once we have multi-level caches
-            log.warn("read: Runtime cache is disabled, but read request is requesting to cache");
+            if (runtime.getParameters().getPriorityLevel() != PriorityLevel.HIGH) {
```

Comment:
On my old review it was suggested to have this log line, I would vote for just removing it and not making this distinction.

Reply:
ok, sounds good. let me remove it then